src/components/Essentials/essentialsfiltersdesktop.js

```diff
@@ -35,7 +35,7 @@ const useInputLabelStyles = makeStyles(() => ({
 const useMenuItemStyles = makeStyles(() => ({
   root: {
     fontFamily: 'archia',
-    fontSize: '11px !important',
+    fontSize: '11.5px !important',
     fontWeight: 600,
     textTransform: 'uppercase',
```

Comment:
Adding extra property `minHeight: '14px',` solves the issue. You can revert fontSize changes. 
Make the same changes in file essentialsfiltersmobile.js as well for the mobile view.